application/web/index.html

```diff
@@ -15,15 +15,15 @@
     <div id='app-wrapper'>
         <sidebar role="navigation" aria-label="Sidebar">
             <!--Sidebar Modules-->
-            <div class="sidebar-module">
+            <div id="sidebar-module" class="sidebar-module">
                 <button class="sidebar-drawer">
                     <img src="./assets/drawer.svg"></img>
                 </button>
-                <button class="new-journal">
+                <button id="new-journal-button" class="new-journal" contentEditable="true" onclick="newJournal()">
                     <span>New Journal</span>
                     <img src="assets/journal-icon.svg" alt="Journal Icon" class="journal-icon">
                 </button>
-                <div class="journal-entry">
+                <div id="journal-entries" class="journal-entry">
                     <span>My Journal</span>
                     <img class='vdots' src='./assets/vdots.svg'></img>
                 </div>
```

Comment:
wouldn't this require there to have storage for the array of titles of journals to display?

Reply:
I thought you don't need to because you can do add an event listener to change "New Journal" to a text field then user input some stuff, then clicks enter as another event listener, but if it's too complicated to do, we can leave that for the next sprint

Reply:
Oh and then not locally store, yeah this was a misunderstanding on my part. I fixed the other issues I believe, but yeah next sprint I can try to tackle this properly.